### PR TITLE
ISSD-1822 [CP 2.0] Release Notes page - 1st tab: Release Highlights

### PR DESCRIPTION
**Pull request by samuelleolopes**
[ISSD-1822](https://liferay.atlassian.net/browse/ISSD-1822)

### CODE REVIEW

**Review by kylembischof — CHANGES_REQUESTED** [review]
when i run `gw formatSource && yarn format` it lists out some errors. can you check these?

```
/home/me/dev/projects/liferay-portal/modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/release-notes-redirect/index.js
  6:18  error  'layoutMode' is not defined.  no-undef
  7:21  error  'layoutMode' is not defined.  no-undef

/home/me/dev/projects/liferay-portal/modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/release-notes-tabs/index.js
  6:18  error  'fragmentElement' is not defined.  no-undef
  7:24  error  'fragmentElement' is not defined.  no-undef
  8:18  error  'layoutMode' is not defined.  no-undef
  14:2  error  'fragmentElement' is not defined.  no-undef
  15:34  error  'fragmentNamespace' is not defined.  no-undef
  20:2  error  'fragmentElement' is not defined.  no-undef
  21:34  error  'fragmentNamespace' is not defined.  no-undef
  26:7  error  'configuration' is not defined.  no-undef

✖ 10 problems (10 errors, 0 warnings)
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-dropdown/ddm-template.ftl [hunk 1]
```diff
@@ -63,4 +63,40 @@
...
+		}
+
+		const newURL = currentURL + 'v/' + value;
+		window.location.href = newURL;
+	}
+
+	let value = 'RELEASE-NOTES-HIGHLIGHT-STRUCTURE';
+	document.addEventListener('DOMContentLoaded', function() {
+		document.querySelectorAll('.active').forEach(function(elemento) {
+				value = getValueForElementId(elemento.id);
```
[the remark this answers is not in the document]
Comment (samuelleolopes) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-dropdown/ddm-template.ftl:94]: Corrected to "element". thanks

**Comment by samuelleolopes** [comment]
> when i run `gw formatSource && yarn format` it lists out some errors. can you check these?
> 
> ```
> /home/me/dev/projects/liferay-portal/modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/release-notes-redirect/index.js
>   6:18  error  'layoutMode' is not defined.  no-undef
>   7:21  error  'layoutMode' is not defined.  no-undef
> 
> /home/me/dev/projects/liferay-portal/modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/release-notes-tabs/index.js
>   6:18  error  'fragmentElement' is not defined.  no-undef
>   7:24  error  'fragmentElement' is not defined.  no-undef
>   8:18  error  'layoutMode' is not defined.  no-undef
>   14:2  error  'fragmentElement' is not defined.  no-undef
>   15:34  error  'fragmentNamespace' is not defined.  no-undef
>   20:2  error  'fragmentElement' is not defined.  no-undef
>   21:34  error  'fragmentNamespace' is not defined.  no-undef
>   26:7  error  'configuration' is not defined.  no-undef
> 
> ✖ 10 problems (10 errors, 0 warnings)
> ```

Resolved with "Liferay.<element>"

**Comment by samuelleolopes** [comment]
> when i run `gw formatSource && yarn format` it lists out some errors. can you check these?
> 
> ```
> /home/me/dev/projects/liferay-portal/modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/release-notes-redirect/index.js
>   6:18  error  'layoutMode' is not defined.  no-undef
>   7:21  error  'layoutMode' is not defined.  no-undef
> 
> /home/me/dev/projects/liferay-portal/modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/release-notes-tabs/index.js
>   6:18  error  'fragmentElement' is not defined.  no-undef
>   7:24  error  'fragmentElement' is not defined.  no-undef
>   8:18  error  'layoutMode' is not defined.  no-undef
>   14:2  error  'fragmentElement' is not defined.  no-undef
>   15:34  error  'fragmentNamespace' is not defined.  no-undef
>   20:2  error  'fragmentElement' is not defined.  no-undef
>   21:34  error  'fragmentNamespace' is not defined.  no-undef
>   26:7  error  'configuration' is not defined.  no-undef
> 
> ✖ 10 problems (10 errors, 0 warnings)
> ```


Hi @kylembischof, It fixed with /* eslint-disable no-undef */, thanks @jairmedeiros

**Comment by kylembischof** [comment]
@jairmedeiros can you please take a look when you have a chance. thanks!

**Comment by jairmedeiros** [comment]
@samuelleolopes @pedroliveirav I scheduled a meeting tomorrow to discuss about this PR, thanks.

**Review by pedroliveirav — CHANGES_REQUESTED** [review]

**Comment by samuelleolopes** [comment]
PR reopened https://github.com/liferay-customer-delivery/liferay-portal/pull/64

**Pull request closed without merging**